pymc3/gp/cov.py

```diff
@@ -353,6 +353,15 @@ class Periodic(Stationary):
 
     .. math::
        k(x, x') = \mathrm{exp}\left( -\frac{\mathrm{sin}^2(\pi |x-x'| \frac{1}{T})}{2\ell^2} \right)
+
+    The interpretation of length-scale for this kernel is different than the
+    one used normally (see [1]_). Priors on length scale must be put keeping
+    this in mind.
```

Comment:
```suggestion
    The scaling factor for this kernel is different compare to the other standard
    definition (see [1]_), thus change the interpretation of length-scale in a subtle
    way. Please keep that in mind when you are defining priors on length scale,
    especially when you are trying to replicate other results.
    
    You can divided the length-scale by 2 in the kernel initialization to recover the
    standard definition.
```

@bwengals is this correct?

Reply:
If I read that it's different, how should one interpret it?  I think it'd be good to be specific.  

Also small wording suggestion, how does this sound?

The scaling factor for this kernel is different compare**d** to the more common definition (see [1]_).  Here 0.5 is in the exponent instead of the more common value, 4.  Divide the length-scale by 2 in the kernel initialization to recover the standard definition.

Reply:
Sorry for not replying earlier. I missed @bwengals's reply.

> Here, 0.5 is in the exponent instead of the more common value, 4.

I think what you meant here was:

Here, 0.5 is in the exponent instead of the more common value, 2.

Right?

Reply:
whoops, yep you're right!